### ISSUE TITLE
Request: Different volume mapping options

### ISSUE THREAD

**Issue by tom472** [binhex/arch-delugevpn#11]
binhex - This image is awesome and works great with my PIA account, plus the additional add-on of Privoxy is extremely useful.

I have a question / request though:

Would it be possible to re-work this image a bit to provide for two volume mapped folder locations instead of just the one `/data` location?

Coming from a non-container version of Deluge the general recommendation is to have/use a two folder set-up generally something like `~/downloading` and `~/complete`.
I would like to keep those two folders in use and volume map them into this container but right now there is only the `/data` location available (not including `/config` which should remain on its own).

Anyway just a thought - would really appreciate it if the tweak could be made - but if not I'll find some way to work around it.

Thanks again for all your awesome Docker images.

**Comment by nbisson** [comment]
Agreed. I'm trying to map an external volume as my completed folder but can't since the /data folder must remain on the volume where deluge is hosted

**Comment by binhex** [comment]
Hi guys you do know you can specify as many volume mappings as you like right? Just simply add in more -v flags to the docker run command if you wish to map in additional volumes.

**Comment by tom472** [comment]
I know more -v flags can be added - but doesn't the counterpart location need to exist in the base image before it can be mapped to a host location.
If `~/downloading` and `~/complete` don't already exist in the image - I can't map them to host locations when starting the container.

That is unless I'm wrong - and docker will create those locations "on the fly" if they do not exist in the image when the container is started.

i.e. in your Dockerfile you are define these volumes:
`# map /config to host defined config path (used to store configuration from app)
VOLUME /config

# map /data to host defined data path (used to store data from app)
VOLUME /data`

Wouldn't the additional volume locations need to be defined too?

Thanks

**Comment by binhex** [comment]
In a word no, your don't need to create the container side of the volume mapping, it does indeed create it on the fly. The other better option is just to create some structure under the/data folder so something like this:-

/data/incomplete/
/data/completed/

Then specify this in deluge

**Comment by tom472** [comment]
Wow - that is something new i just learned with Docker - that it will create the folders on the fly. 
That is actually neat to know / learn.

Your suggestion is actually the path I ended up taking making the folders inside the \data folder.
Had to tweak a couple other apps to point to the new location - but I got it worked out.

Thanks for jumping in an commenting on this - and again for all the great work on your Docker images.
